### ISSUE TITLE
mathjax CDN has been retired; need to migrate to another CDN and must use https to prevent mixed-mode blocking errors

### ISSUE THREAD

**Issue by nathanv** [wet-boew/wet-boew#8078]
Currently, when rendering a site with MathML on https, the current use of the mathjax CDN has a number of problems.
1. Does not load because of [mixed-mode blocking](https://developer.mozilla.org/en-US/docs/Web/Security/Mixed_content/How_to_fix_website_with_mixed_content) as seen here: https://wet-boew.github.io/wet-boew/demos/mathml/mathml-en.html . The offending line is here: https://github.com/wet-boew/wet-boew/blob/master/src/core/wb.js#L531 :
`

					Modernizr.load( {
						load: "http://cdn.mathjax.org/mathjax/latest/MathJax.js?config=Accessible",
						complete: function() {

							// Identify that initialization has completed
							wb.ready( $document, componentName );
						}
					} );`
2. points to an http version
Further, console says: "WARNING: cdn.mathjax.org has been retired. Check https://www.mathjax.org/cdn-shutting-down/ for migration tips."
3. The "accessible" configuration is deprecated as seen here: http://docs.mathjax.org/en/latest/config-files.html#the-accessible-configuration-file-deprecated
4. Latest.js is used instead of a specific version (which is not recommended anymore) as per notes here: https://www.mathjax.org/cdn-shutting-down/

Suggested resolution: switch to a different CDN such as https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js or something else agreed upon by wet-boew maintainers. In principle, I'd prefer http://www.jsdelivr.com/ but I have an [outstanding](https://jsdelivr.uservoice.com/forums/164147-plugins-submission/suggestions/19875577-mathjax-had-have-the-versions-load-properly-throug) [support requests that needs to be resolved](https://jsdelivr.uservoice.com/forums/164147-plugins-submission/suggestions/2967289-mathjax-http-www-mathjax-org) before we can go with it.

**Comment by nschonni** [comment]
Probably also a good idea to add a local fallback. I believe this was avoided in the past because the library is large.

**Comment by MartinKolarik** [comment]
@nathanv mathjax is available at https://cdn.jsdelivr.net/npm/mathjax@2.7.1/

**Comment by nathanv** [comment]
I figured out a way to do fail-overs as seen in the above PR. The first thing I did was have a relatively quick timeout trying to load the CDN. Without the timeout limit, it takes way too long to load the fail-over.
Secondly, I loaded MathJax locally using bower. Yes MathJax is heavy but when it builds, I'm pulling in less than everything for the local fail over. I tried it out on the sample until I got no more errors posting.
Since we're using bower, I didn't end up using MathJax-grunt-cleaner and simply pulled ideas from it.
